shared/message/input/MotionCapture.proto

```diff
@@ -34,6 +34,7 @@ message MotionCapture {
         uint32 id       = 1;
         fvec3  position = 2;
         float  size     = 3;
+        uint32 modelId = 4;
```

Comment:
```suggestion
        uint32 model_id = 4;
```
[we use snake_case ](https://nubook.nubots.net/guides/general/code-conventions#page-content)